### ISSUE TITLE
Color contrast ratio of the focus indication over Select IoT Hub list item is less than 3:1.

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#126897]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yWCAG;#DesktopApp;#Visual Studio Code Client;#ColorContrast;#WCAG1.4.11;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;

### Environment details:
Application Name: Azure Stream Analytics Tools for Visual Studio Code
Operating system: Windows 10 
Visual studio code version: 1.55.2(User Setup)

### Repro Steps:

1. Launch Visual Studio code application with installed Azure Stream Analytics Tools extension. 
2. Tab to the "Explorer" section in activity bar. 
3. Press control+shift +P and edit bar will appear. 
4. Type "ASA:Create new project" and press enter to create project. 
5. New project will get created. 
6. Tab to Input item in Project Tree Section 
7. Right click on input and navigate to "ASA : Add Input" using arrow key and press enter. 
8. Select IoT hub from available option using enter key. 
9. Select "select from azure subscriptions" from the available option using enter key. 
10. Type any name for input file and press enter to create Input json file. 
11. Tab to "Select IoT Hub" button.
12. Open AI4D and Check whether Color contrast ratio of the focus indication over Select IoT Hub list item is less than 3:1. or not.
13. 

### Actual Result:
Color contrast ratio of the focus indication over Select IoT Hub list item is less than 3:1.
Note: - This issue is observed in throughout application for any list item.

### Expected Result:
The color contrast ratio of the focus indication over Select IoT Hub list item should be greater than or equal to 3:1.

### User Impact:
The low vision users may not able to see the focus over Select IoT Hub list item.

### Attachment:
[A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_SelectIoTHub_NonTextContrast.zip](https://github.com/microsoft/vscode/files/6693670/A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_SelectIoTHub_NonTextContrast.zip)

**Comment by misolori** [comment]
This was already [fixed in 1.57](https://code.visualstudio.com/updates/v1_57#_updated-quick-pick-suggest-widget-colors) and it appears that you are on 1.55 so please try again on the latest version. Thanks.
